app/views/admin/badge_achievements/index.html.erb

```diff
@@ -36,7 +36,7 @@
         <td class="justify-content-center">
           <img class="mx-auto mt-3" width="40" height="40" src="<%= badge_achievement.badge.badge_image %>" alt="badge image">
         </td>
-        <td><%= link_to "Remove", url_for(action: :destroy, id: badge_achievement.id), method: :delete, data: { confirm: "Are you sure?" }, class: "btn btn-danger" %></td>
+        <td><%= link_to "Remove", admin_badge_achievement_path(badge_achievement), class: "crayons-btn crayons-btn--danger", method: :delete, data: { confirm: "Are you sure?" } %></td>
```

Comment:
I've changed most things to use the path helpers which makes it a lot easier to refactor routes